config/default.go

```diff
@@ -17,22 +17,21 @@ port: 0
 proxyPort: 16789
 dnsPort: 26789
 debug: false
-disableANSI: false
 disableTele: false
+inDocker: false
 generateGithubActions: false
 containerName: ""
 networkName: ""
 buildDelay: 30
 test:
   selectedTests: {}
-  ignoredTests: {}
```

Comment:
why is this removed ? please go through the config.go and update according to that we still use this flag